### PR TITLE
Test article bulk import in end2end enviromment

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

**Comment by nlisgo** [comment]
Have just restarted the build on travis.

File: Jenkinsfile [hunk 1]
```diff
@@ -15,6 +15,7 @@ elifePipeline {
             {
                 builderDeployRevision 'journal-cms--end2end', commit
                 builderSmokeTests 'journal-cms--end2end', '/srv/journal-cms'
+                builderCmd 'journal-cms--end2end', 'cd /srv/journal-cms; drush article-import-all --limit=25'
```
Comment (nlisgo) [inline Jenkinsfile]: in order for drush to run contextually it should be run from `/srv/journal-cms/web` and we should then reference the `drush` command in the the `vendor/bin` folder.

```
builderCmd 'journal-cms--end2end', 'cd /srv/journal-cms/web; ../vendor/bin/drush article-import-all --limit=25'
```

Comment (giorgiosironi) [inline Jenkinsfile]: Right :+1:

**Comment by giorgiosironi** [comment]
If we will do this, we don't do it in end2end.

**Pull request closed without merging**